modules/repository-datastax/db/mutation/V036__Alter_Experiment_Add_Tags.cql

```diff
@@ -0,0 +1 @@
+alter table experiment ADD tags set<text>;
```

Comment:
New location for cassandra deltas: modules/repository-datastax/src/main/resources/com/intuit/wasabi/repository/impl/cassandra/migration